### PR TITLE
Phoneme distribution analysis tutorial

### PR DESCRIPTION
**Pull request by styagi130**

### CODE REVIEW

File: tts-python-phoneme-distribution.ipynb [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+    "\n",
+    "pp.pprint(data_combined)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ae847e5",
+   "metadata": {},
+   "source": [
+    "### Phoneme comparison to a reference distribution \n",
```
Comment (blisc) [inline tts-python-phoneme-distribution.ipynb:361]: Can you add the final plot?

File: tts-python-phoneme-distribution.ipynb [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+   "id": "9e6ad9a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Put all the imports here for ease of use, and cleanliness of code \n",
+    "import os\n",
+    "import pprint\n",
+    "import json\n",
+    "# CMU dict for phoneme distribution \n",
+    "import cmudict\n",
```
Comment (blisc) [inline tts-python-phoneme-distribution.ipynb:45]: Can you use Nemo instead?

File: tts-python-phoneme-distribution.ipynb [hunk 3]
```diff
@@ -0,0 +1,509 @@
...
+    "                freqs.update(phonemes.get(word, [])) \n",
+    "    return freqs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c962af3",
+   "metadata": {},
+   "source": [
+    "### Sample copurs Phonemes\n",
```
Comment (jasro23) [inline tts-python-phoneme-distribution.ipynb:212]: Fix the typo for corpus

File: tts-python-phoneme-distribution.ipynb [hunk 4]
```diff
@@ -0,0 +1,573 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6d17e187",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/stable/scripts/tts_dataset_files/cmudict-0.7b_nv22.08"
```
Comment (blisc) [inline tts-python-phoneme-distribution.ipynb:83]: Let's pin the nemo version, not `stable`

File: tts-python-phoneme-distribution.ipynb [hunk 5]
```diff
@@ -0,0 +1,573 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f1c6756b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a list of the CMU phoneme symbols\n",
+    "phonemes = {k: v[0] for k, v in g2p.phoneme_dict.items()}\n",
+    "pp.pprint(list(g2p.phoneme_dict.items())[:10])"
```
Comment (blisc) [inline tts-python-phoneme-distribution.ipynb:132]: Why do we have this magic number `[:10]`

File: tts-python-phoneme-distribution.ipynb [hunk 6]
```diff
@@ -0,0 +1,573 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b6d3f434",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ref_corpus_file = \"text_files/phoneme-dist/reference_corpus.json\"\n",
+    "f = open(ref_corpus_file)\n",
+    "freqs_ref = json.load(f)\n",
+    "f.close()"
```
Comment (blisc) [inline tts-python-phoneme-distribution.ipynb:155]: Use a with statement instead of open, close

File: tts-python-phoneme-distribution.ipynb (hunk 5 above)
Comment (styagi130) [reply tts-python-phoneme-distribution.ipynb:132]: Its an accident. I has stayed from when I just looking at the data. I will remove this.

File: tts-python-phoneme-distribution.ipynb [hunk 7]
```diff
@@ -0,0 +1,564 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cdfc8c0",
+   "metadata": {},
+   "source": [
+    "#### Get arpabet file\n",
+    "\n",
+    "Get the CMU grapheme to phoneme dictionary from [nemo](https://raw.githubusercontent.com/NVIDIA/NeMo/stable/scripts/tts_dataset_files/cmudict-0.7b_nv22.08)."
```
Comment (blisc) [inline tts-python-phoneme-distribution.ipynb:73]: Can you update this link as well?

**Review by blisc — APPROVED** [review]

**Pull request merged**